plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -59,12 +59,12 @@ def launch(
     assert launcher.task_function is not None
 
     setup_commands = launcher.env_setup.commands
+    packages = filter(
+        lambda x: x[1] is not None, launcher.env_setup.pip_packages.items()
+    )
```

Comment:
why would this be `None`?

Reply:
This is a new feature:
it's a way to remove an existing a package from the packages dict.